OptionSelector/CQDetailTableViewController.m

```diff
@@ -0,0 +1,119 @@
+//
+//  CQDetailTableViewController.m
+//  Unit1WeekFourHomework
+//
+//  Created by Jackie Meggesto on 8/12/15.
+//  Copyright (c) 2015 Jackie Meggesto. All rights reserved.
+//
+
+#import "CQDetailTableViewController.h"
+
+@interface CQDetailTableViewController ()
+@property (nonatomic) NSIndexPath *checkedIndexPath;
+
+
+@end
+
+@implementation CQDetailTableViewController
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    
+    
+    
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+}
+
+-(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+    self.checkedIndexPath = indexPath;
+    self.category.cqIndexPath = indexPath;
+    self.category.selection = self.category.options[indexPath.row];
+    [self.tableView reloadData];
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+#warning Potentially incomplete method implementation.
```

Comment:
Style: You can safely remove these `#warning` directives. They're just there as a reminder for you to implement these methods if needed. In this case, you can actually remove this method altogether, because table views default to having 1 section.